atd-vzd/migrations/default/1697646059138_update_fatality_soft_delete_function/up.sql

```diff
@@ -0,0 +1,60 @@
+CREATE OR REPLACE FUNCTION public.update_fatality_soft_delete()
+ RETURNS trigger
+ LANGUAGE plpgsql
+AS $function$
+DECLARE
+    cris_id integer; -- used to hold vz id when triggered from cris_facts schema
+    injury_severity_id integer;
+    BEGIN
+    RAISE NOTICE 'Here we are.';
+        -- if updated table was primary person
+    	IF (TG_TABLE_NAME = 'atd_txdot_primaryperson') THEN
```

Comment:
is the plan to at a later date add in the `TG_TABLE_SCHEMA = 'cris_facts'` or `TG_TABLE_SCHEMA = 'vz_facts'` parts of the if else, like you did with the persons table?